cairo/ethereum/cancun/state.cairo

```diff
@@ -930,6 +929,29 @@ func destroy_touched_empty_accounts{poseidon_ptr: PoseidonBuiltin*, state: State
     );
 }
 
+func empty_transient_storage{range_check_ptr}() -> TransientStorage {
+    let (dict_ptr) = default_dict_new(0);
```

Comment:
default is not zero, it's a pointer to a U256Struct(0,0)

Reply:
can we create an issue to check?

Reply:
Done https://github.com/kkrt-labs/keth/issues/780